external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala

```diff
@@ -74,7 +74,7 @@ private[kafka010] class KafkaSourceRDD(
   }
 
   override def getPartitions: Array[Partition] = {
-    offsetRanges.zipWithIndex.map { case (o, i) => new KafkaSourceRDDPartition(i, o) }.toArray
+    offsetRanges.zipWithIndex.map { case (o, i) => KafkaSourceRDDPartition(i, o) }.toArray
```

Comment:
I'm OK with this too, though for case classes it really doesn't matter which way it's called, and we don't do it consistently either way